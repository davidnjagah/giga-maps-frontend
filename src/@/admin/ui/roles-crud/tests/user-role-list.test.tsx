import { render } from '@testing-library/react';
import { testWrapper } from '~/tests/jest-wrapper';
import RolesList from '../user-roles-list.view';

describe('User role', () => {
  it('should render component', () => {
    const { asFragment } = render(testWrapper(<RolesList />));
    expect(asFragment).toMatchSnapshot();
  });
});
