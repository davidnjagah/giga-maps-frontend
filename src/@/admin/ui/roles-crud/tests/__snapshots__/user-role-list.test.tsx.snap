// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User role should render component 1`] = `[Function]`;
